plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -343,17 +344,22 @@ private static DeltaLakeColumnMetadata mapColumn(TypeManager typeManager, JsonNo
     }
 
     public static Map<String, String> getColumnComments(MetadataEntry metadataEntry)
+    {
+        return getColumnProperties(metadataEntry, DeltaLakeSchemaSupport::getComment);
+    }
+
+    public static <T> Map<String, T> getColumnProperties(MetadataEntry metadataEntry, Function<JsonNode, T> extractor)
```

Comment:
It would be better to move these `getColumnProperties()` after `getInvariants()`